gbalink/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 include_directories(. ./../Runtime ${ATHENA_INCLUDE_DIR} ${JBUS_INCLUDE_DIR})
-add_executable(gbalink main.cpp)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ldl")
+add_executable(gbalink main.cpp ../hecl/extern/boo/logvisor/lib/logvisor.cpp ../hecl/lib/hecl.cpp)
```

Comment:
I'm going to spin off the hecl::SystemChar type to a dedicated header. That should avoid having to link these extra objects.